### PR TITLE
Added TED CLI section to topaz_cli.md.

### PR DESCRIPTION
**Pull request by ratishkr**
Updates for new feature TED CLI.

### CODE REVIEW

**Comment by ratishkr** [comment]
Incorporated comments from PR 48 and resolved the issue with file space issue.
I closed that PR, and it gave the option to delete that branch, and with that went my PR comment as well, so re-entering here.

The issue with the removal of trailing space automatically happened when he file is saved using atom editor.  And the duplicate line is actually a missing part of a line change, when I tried doing discard like how I used to do it in bitbucket. I thought I discarded all those changes, but apparently only one line of change was discarded!

I used notepad++ to make the change this time.

File: docs/apis/topaz_cli.md [hunk 1]
```diff
@@ -933,4 +933,91 @@ Parameter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Abbr.  | Description
...
\ No newline at end of file
+`-save` | -s | Save output or not. Set to TRUE to write output, or FALSE for no output. When set to FALSE, no files will be created and the archive will not be updated. Defaults to TRUE.
+
+## Topaz Enterprise Data CLI (TedCLI.bat)
+
+The Topaz Enterprise Data CLI executes TED specifications such ConverterPro, ComparePro, Related Extract, Related Loader and Execution Suites. The specifications will have to be created using the relevant application client available in the File-AID/EX perspective of Topaz and the specifications should be saved and made available in any of the shared repositories. Moreover the shared repository should be configured on a standalone communication manager for using it with TED CLI. Further, to execute the specifications via the TED CLI, standalone execution server needs to be installed if not already available. Please note that the embedded communication manager and execution server that is available with the Topaz TED feature cannot be used for executing TED CLI.
```
Comment (bszydlow) [inline docs/apis/topaz_cli.md:940]: "TED specifications such ConverterPro..."
including instead of such would be better

Comment (ratishkr) [reply docs/apis/topaz_cli.md]: Made that one plus additional changes to the content.

**Review by bszydlow — APPROVED** [review]

**Comment by ratishkr** [comment]
Merged to release/20.6 instead.

**Pull request closed without merging**